### DISCUSSION TITLE
Lenguage selection

### DISCUSSION THREAD

**Discussion by jlcases** [assafelovic/gpt-researcher#141] [General]
I put the query in Spanish but reports are in English. 
Can I change that? 
Thx

**Comment by jlcases** [comment]
Greetings,

I have developed an enhanced feature for multi-language support:

Language selection capability, supporting up to 7 different languages.
<img width="1274" alt="image" src="https://github.com/assafelovic/gpt-researcher/assets/69396384/cac984da-88c1-4785-aa6d-b1a13308e08a">

Language Selection Interface

Implementation of 8 queries: 4 in English and 4 in the user-selected language to optimize search results.

Generation of reports in the user-selected language.
<img width="1062" alt="image" src="https://github.com/assafelovic/gpt-researcher/assets/69396384/1304928a-0df5-45f4-97a7-11ae8a9e8ec6">


Report Interface

I would be honored to contribute this feature to your project and am open to further enhancing it based on your feedback.

Best regards,

Jose Luis Cases

**Reply by GreaseMonkey88** [reply]
Has language selection been removed? I cannot find it anymore.
